docs_src/stores.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const sourceType = writable('html');
```

Comment:
Hvad siger du til:
```
import { Writable, writable } from 'svelte/store';

export const sourceType: Writable<'svelte' | 'html'> = writable('html');
```

Reply:
Vi skrev dette i dogmer:  _Writables må kun opdateres vha. actions og må ikke exporteres direkte_ . Man kan dog argumentere for, det er så simpelt her, så det er overkill. Hvad siger du @tegner ?

Reply:
tænker også at det er overkill at gøre for meget ud af det der